test/unit/org/apache/cassandra/tools/nodetool/ClientStatsTest.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.tools.nodetool;
+
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import com.datastax.driver.core.ResultSet;
+import org.apache.cassandra.cql3.CQLTester;
+import org.apache.cassandra.tools.ToolRunner;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ClientStatsTest extends CQLTester
+{
+    @BeforeClass
+    public static void setup() throws Throwable
+    {
+        requireNetwork();
+        startJMXServer();
+    }
+    
+    @Before
+    public void config() throws Throwable
+    {
+        ResultSet result = executeNet("select release_version from system.local");
+    }
+    
+    @Test
+    public void testClientStatsHelp()
+    {
+        ToolRunner.ToolResult tool = ToolRunner.invokeNodetool("help", "clientstats");
+        tool.assertOnCleanExit();
+        
+        String help =   "NAME\n" +
+                        "        nodetool clientstats - Print information about connected clients\n" +
+                        "\n" +
+                        "SYNOPSIS\n" +
+                        "        nodetool [(-h <host> | --host <host>)] [(-p <port> | --port <port>)]\n" +
+                        "                [(-pp | --print-port)] [(-pw <password> | --password <password>)]\n" +
+                        "                [(-pwf <passwordFilePath> | --password-file <passwordFilePath>)]\n" +
+                        "                [(-u <username> | --username <username>)] clientstats [--all]\n" +
+                        "                [--by-protocol] [--clear-history]\n" +
+                        "\n" +
+                        "OPTIONS\n" +
+                        "        --all\n" +
+                        "            Lists all connections\n" +
+                        "\n" +
+                        "        --by-protocol\n" +
+                        "            Lists most recent client connections by protocol version\n" +
+                        "\n" +
+                        "        --clear-history\n" +
```

Comment:
Nice test!. If you add a test case for `--clear-history`, despite not being directly related to the scope of this ticket, you'd have a complete `clientstats` test class which I think it's a quick win. Wdyt?